src/navigation/MainTabNavigator.tsx

```diff
@@ -95,7 +77,7 @@ const MainTabNavigator: FunctionComponent = () => {
         size={size}
       />
     )
-    return userHasNewExposure ? applyBadge(tabIcon) : tabIcon
```

Comment:
I thought this was for another feature we were waiting for? Is it going away?

Reply:
Perhaps @johnschoeman can comment — I think the development of that feature is sufficiently far out that it doesn’t make sense to have half-developed code lying around.

Reply:
Havent seen much of this feature recently, so i would lean towards considering this dead code, and therefore we should remove it.